cmd/gsa/entry.go

```diff
@@ -37,7 +37,7 @@ func entry() error {
 			return err
 		}
 	} else {
-		writer = utils.Stdout
```

Comment:
please set this to utils.Stderr

Reply:
Thank you for the feedback. However, since the writer is intended for program results, I believe it is more appropriate to use Stdout rather than Stderr.

```
./gsa ./gsa | less
```

Alternatively, would it be better to define utils.Stdout similarly to utils.Stderr?

Reply:
Yes, I think it would be better to define the same logic for Stdout as well, thanks for your contribution!